lib/logflare_web/controllers/api/fallback_controller.ex

```
@@ -10,6 +10,12 @@ defmodule LogflareWeb.Api.FallbackController do
     |> json(%{errors: errors})
   end
 
+  def call(conn, {:error, :unauthorized}) do
+    conn
+    |> put_status(401)
+    |> halt()
```

We should return an error message